apps/site/pages/en/learn/typescript/publishing-a-ts-project.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Publishing a TypeScript project
```

Comment:
```suggestion
title: Publishing a TypeScript package
```

Why not using word package ?

Reply:
Ah, yes that's better. Should also change the file name and json refs.